src/groundlight/internalapi.py

```diff
@@ -20,6 +22,14 @@ class NotFoundError(Exception):
     pass
 
 
+class InspectionError(Exception):
```

Comment:
When does this get raised?  Can we add a comment to that effect?  Also, it should probably inherit from something other than `Exception`, like `RuntimeError`.  Or better-yet one of our own API exception classes.